python/desc/imsim/camera_readout.py

```diff
@@ -448,3 +473,205 @@ def set_phosim_bboxes(amp):
     amp.setRawPrescanBBox(afwGeom.Box2I(afwGeom.Point2I(0, 1),
                                         afwGeom.Extent2I(4, 2000)))
     return amp
+
+PixelParameters = namedtuple('PixelParameters',
+                             ('''dimv dimh ccdax ccday ccdpx ccdpy gap_inx
+                                 gap_iny gap_outx gap_outy preh'''.split()))
+pixel_parameters \
+    = {'E2V': PixelParameters(2002, 512, 4004, 4096, 4197, 4200, 28,
```

Comment:
Should these file level namespace objects be defined at the top of the file? Or is it better to have them closer to the function where you used them?

Also, I'm probably just not understanding, but rather than hardcoding in the pixel parameters, should these values be set from the parameters in the segmentation.txt file available in the amp object (same comment for the old code above in the set_phosim_bboxes function).

Reply:
I hard-coded these here because **they** **will** **never** **change** since they are part of the vendors' designs and those are fixed.   It didn't seem worth the effort to make put this in segmentation.txt (and alter the format from phosims) or make up some new configuration file for something that will never change.